rules-tests/TypeDeclaration/Rector/ClassMethod/AddVoidReturnTypeWhereNoReturnRector/Fixture/throws_method.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector\Fixture;
+
+final class ThrowsMethod
+{
+    protected function getValues()
+    {
+        if (rand(0,1)) {
+            throw new \Exception();
+        }
+
+        echo 'hello world';
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector\Fixture;
+
+final class ThrowsMethod
+{
+    protected function getValues(): void
```

Comment:
this is the bug I am trying to solve. a method, which contains a non-top-level never type, should get the `void` return type as it will not get a never type

https://3v4l.org/gNmVO